src/main/java/org/wildfly/security/auth/realm/FileSystemSecurityRealm.java

```diff
@@ -164,6 +165,11 @@ public ModifiableRealmIdentity getRealmIdentityForUpdate(final IdentityLocator l
         return getRealmIdentity(locator, true);
     }
 
+    @Override
+    public void registerIdentityChangeListener(Consumer<IdentityLocator> listener) {
+        // no need to register the listener given that changes to identities are done through the realm
```

Comment:
I don't see where changes (done through the realm) are caught by listeners though.